src/index.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import './App.css';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 import 'bootstrap/dist/css/bootstrap.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```

Comment:
- [ ] I would suggest you that your App.js(main component) should have the imported Calculator.js component and the `App` should be used here in index.js as it is generated by React.